devito/types/relational.py

```diff
@@ -204,7 +200,7 @@ class Ne(sympy.Ne):
     `Ne` may be used to express a relation.
 
     >>> from devito import Grid, Function, ConditionalDimension, Eq, Operator
-    >>> from devito.types import Ne
+    >>> from devito import Ne
```

Comment:
forgot this one, should be merged with first line

Reply:
Fixed